content/Software Support/IDE Settings/About-mdns-discovery.md

```diff
@@ -3,7 +3,7 @@ title: "About mdns-discovery"
 id: 4506515275548
 ---
 
-Arduino IDE includes an [mDNS discovery tool](https://github.com/arduino/mdns-discovery) which is used to find available network ports.
+Arduino IDE includes an [mDNS discovery tool](https://github.com/arduino/mdns-discovery) which is used to find available network ports. These ports are essential for enabling communication between the IDE and connected devices, particularly for tasks like over-the-air (OTA) uploads. For example, the Board Manager and Library Manager rely on network ports to detect and manage Arduino boards and libraries connected to the network, ensuring seamless communication and updates. By using mDNS discovery, the Arduino IDE can dynamically detect the presence of compatible devices, making the development process more efficient and streamlined.
```

Comment:
```suggestion
Arduino IDE includes an [mDNS discovery tool](https://github.com/arduino/mdns-discovery) used to identify the ports of boards that support the Over-The-Air (OTA) feature. This tool helps Arduino IDE communicate with compatible network-connected boards for tasks like OTA uploads. It detects the addition and removal of these ports on the local network, allowing the IDE to keep the list of available ports updated. This ensures that the port listings, such as those found in the _Tools > Port_ menu and the board selector in the IDE, are up-to-date.
```
This version helps explaining that the mDNS tools identifies available network ports, specifically in the context of communication with boards over-the-air.

Reply:
ok thank you so much 🌞